Docs - S3 masking and nav update to S3 page (#11490)
apache/druid: docs/development/extensions-core/s3.md

### S3 authentication methods

You can provide credentials to connect to S3 in a number of ways, whether for [deep storage](#deep-storage) or as an [ingestion source](#reading-data-from-s3).

> There is a defined order of precedence, as given below.  This means, for example, if you would like to use profile information given in `~/.aws.credentials`, do not set `druid.s3.accessKey` and `druid.s3.secretKey` in your Druid config file as this would take precedence.

Reviewer: ```suggestion
Thee configuration options are listed in order of precedence.  For example, if you would like to use profile information given in `~/.aws.credentials`, do not set `druid.s3.accessKey` and `druid.s3.secretKey` in your Druid config file because they would take precedence.
```
